src/main/java/org/noteskeeper/server/HelloWorldController.java

```diff
@@ -0,0 +1,21 @@
+package org.noteskeeper.server;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+@RequestMapping(path="/api")
```

Comment:
For most annotation there is a more convenient way to put parameter if it is only one and is mapped to 'value' parameter. In that case it is not obligatory to add 'path=', you may just leave '/api'. It might be confusing first time, you just need to understand how does it work.